Modules/Nonunit/Review/test/itkMultiphaseDenseFiniteDifferenceImageFilterTest.cxx

```diff
@@ -105,8 +106,15 @@ itkMultiphaseDenseFiniteDifferenceImageFilterTest(int, char *[])
 
   auto filter = FilterType::New();
 
-  std::cout << "GetNameOfClass() = " << filter->GetNameOfClass() << std::endl;
-  filter->Print(std::cout);
+  ITK_EXERCISE_BASIC_OBJECT_METHODS(
+    filter, MultiphaseDenseFiniteDifferenceImageFilterTestHelper, MultiphaseDenseFiniteDifferenceImageFilter);
 
+
+  unsigned int reinitializeCounter = 1;
+  filter->SetReinitializeCounter(reinitializeCounter);
+  ITK_TEST_SET_GET_VALUE(reinitializeCounter, flter->GetReinitializeCounter());
+
+
+  std::cout << "Test finished." << std::endl;
```

Comment:
Most commits of your PR look fine to me, Jon, thanks. I'm just wondering, is it really necessary for each test to have a "test ending message" printed to standard output?

Reply:
I cannot retrieve the exact PR that prompted this habit (it was more than 5+ years ago), but the summary is that when inspecting the trace of some failing test in the dashboard I was not able to tell whether the test had ended.

Reply:
Thanks for your reply, Jon. Looking at [one of the log files](https://dev.azure.com/itkrobotlinux/ITK.Linux/_build/results?buildId=8845&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=f58be928-45a1-58e7-c1ad-04869565b3f4&l=9202) of the latest commit (8454e9da2df3cde278fcb14b1687d5fee0e3f886), it has lines like:   

> `1/2919 Test #1795:` itkAdditiveGaussianNoiseImageFilterTest ................................................................   Passed    0.06 sec

Just wondering, shouldn't that be sufficient?

Reply:
Niels, I'm well aware that such lines are printed. Believe me. And as said, even the trace in the dashboard was not enough to know the state of the test.

Reply:
If the message is broadly useful, perhaps is should be added to the itkTestDriver code and not each test individually?

Reply:
I agree with that Brad. That deserves some thinking and a separate PR in any case.

Reply:
I know well the problem this tries to solve. A test fails with some error code, and if there is no test output, it is hard to tell whether the failures is due to a DLL being absent, DLL being wrong version, a crash in some library ITK uses, a call to `exit()` or `terminate()` by a dependent library, a crash in itkTestDriver code (after the test itself has finished), a crash in some other global cleanup code, or something else. When a message is printed at the end of a test, that at least allows immediate narrowing down of a problem to "before the end of the test" or after.

Reply:
Thanks for making this more clear 🎯  @dzenanz. Resolving.